lib/common/streams/batchoperation.js

```diff
@@ -112,25 +114,27 @@ BatchOperation.prototype.IsWorkloadHeavy = function() {
   if(this.enableReuseSocket && !this.callInOrder) {
     sharedRequest = 2;
   }
-  return this._activeOperation >= sharedRequest * this.concurrency ||
-    this._isLowMemory() ||
-    (this._activeOperation >= this.concurrency && this._getApproximateMemoryUsage() > 0.5 * SystemTotalMemory);
+  return this._activeOperation >= sharedRequest * this.concurrency ||this._isLowMemory();
 };
 
 /**
-* get the approximate memory usage for batch operation
+* Get the approximate memory usage for batch operation.
 */
 BatchOperation.prototype._getApproximateMemoryUsage = function() {
-  var currentUsage = process.memoryUsage().rss;
+  var currentUsage = azureutil.isBrowser() ? 0 : process.memoryUsage().rss; // Currently, we cannot get memory usage in browsers
```

Comment:
`0` doesn't show the truth. If we cannot get the memory usage in browsers, we can set a minimum required value from experience.